docker-compose-dev.yml

```diff
@@ -22,9 +22,9 @@ services:
       - TEST_DATABASE_URL=mysql2://root:secret@mariadb/test?encoding=utf8
 
   mailcatcher:
-    image: aboutsource/mailcatcher
+    image: tophfr/mailcatcher
     ports:
-      - "1080:1080"
+      - "1080:80"
```

Comment:
depending on what OS you are working with, mapping to port 80 may be restricted.  maybe this is not a problem with docker (does it run with root level permissions).

i would also generally avoid 80 since it is the default http port - and that means someone must not have anything else using it on their machine. 

this is a minor thing.

Reply:
True. But this is about the container-side [port](https://docs.docker.com/compose/compose-file/compose-file-v2/#ports), which moved from 1080 to 80 here. So it will bind to 1080 on the maching running Docker.

Reply:
Thanks for reviewing, btw :)

Reply:
ah, my bad then!  :)  

and yes, thank you for fixing this too.  i haven't tried it out yet, but i know that extra eyes are always useful.